functions/patient_sample_update.py

```diff
@@ -44,7 +44,9 @@ def generate_csvs(event: dict, context: BackgroundContext):
         )
 
 
-_pseudo_blob = namedtuple("_pseudo_blob", ["name", "size", "md5_hash", "time_created"])
+_pseudo_blob = namedtuple(
```

Comment:
Isn't this also defined in the `uploads.py` file? Would it be better to import it from there if so?